src/main/resources/routes

```diff
@@ -3,14 +3,17 @@
 # ~~~~
 
 # Map static resources from the /public folder to the /assets URL path
-GET         /assets/*file                                controllers.Assets.at(path="/public", file)
+GET         /assets/*file                                             controllers.Assets.at(path="/public", file)
 
-GET         /                                            controllers.StartPageController.startPage
-GET         /opportunities                               controllers.OpportunityController.showOpportunities
-GET         /opportunities/guidance/:opportunityId                      controllers.OpportunityController.showGuidancePage(opportunityId:OpportunityId)
-GET         /opportunity/:opportunityId                  controllers.OpportunityController.showOpportunity(opportunityId:OpportunityId, section:Option[Int])
+GET         /                                                         controllers.StartPageController.startPage
+GET         /opportunities                                            controllers.OpportunityController.showOpportunities
 
-GET         /application_form/:id/apply                  controllers.ApplicationController.showOrCreateForForm(id: ApplicationFormId)
+GET         /opportunities/guidance/:opportunityId                    controllers.OpportunityController.showGuidancePage(opportunityId:OpportunityId)
+GET         /manage/opportunities/guidancePM/:opportunityId           controllers.OpportunityController.showPMGuidancePage(opportunityId:OpportunityId)
```

Comment:
Agree. Seems redundant since it's under the `/manage` domain.

Not part of this PR, I know, but `application_form` should probably be `application-form` to fit in with url guidelines as well, no?